src/Location.js

```diff
@@ -2,40 +2,40 @@ import React, { useState, useContext } from 'react'
 import Weather from './Weather'
 import dataContext from './dataContext'
 
-const Location = () => {
-  const { weather, setWeather } = useContext(dataContext)
+const MONTHS = [
```

Comment:
We can move these "global" variables to the top of the file for readability, and mark them as global by uppercasing them. Also, forgot, but we can also alphabetize these. :)